### PR TITLE
Add language selection page (Fixes #5827)

### LINKED ISSUE #5827: SEO - Build a country/language selector page
Build a country/language selector page for better crawlability https://bugzilla.mozilla.org/show_bug.cgi?id=1439377
****
Sketch file: https://mozilla.app.box.com/s/qevzuvmufmag8pnjvn4hv3esfssmlv3y
URL: https://www.mozilla.org/locales/
Title: "Choose your language or locale to browse Mozilla.org
Meta description: "Select your country or region to indicate your preferred language. Learn how Mozilla puts people before profit, making products designed to keep the internet open to all."
****

- [x]  page design

- [x] Locales are hard coded for v.1

- [x]  Protocol ready (no longer new component "table" only change in footer https://github.com/mozilla/protocol/issues/259 needed)

- [x] Meta description (Clay and Raphael)

- [x] Ready to code

- [ ] Page linked in the footer as described https://github.com/mozilla/bedrock/issues/5827#issuecomment-434395964

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a v1 language selection page at `/locales/`.
- List of locales is hard-coded and based on the currently active locales for the www.mozilla.org home page.
- This page will be linked to from the footer. There's a [Protocol PR here](https://github.com/mozilla/protocol/pull/260) to enable that. This doesn't need to block reviewing & merging this page though.

Note: will file an issue to research automation for this list post-launch.

## Issue / Bugzilla link
#5827

## Testing
Demo: https://bedrock-demo-agibson.oregon-b.moz.works/locales/

- [x] Sanity check that locales seem to be in the correct geographical groups?

### CODE REVIEW

File: media/css/mozorg/locales.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+        li {
+            margin-bottom: $spacing-lg;
+        }
+    }
+
+    @media #{$mq-sm} {
+        ul {
+            @include grid-gap($spacing-lg $spacing-xl);
+            display: grid;
```
Comment (alexgibson) [inline media/css/mozorg/locales.scss]: Note for reviewer: as this page is so simple, I decided not to include a float/flex based fallback for browsers that don't support grid. Older browsers will just get a simple vertical list. I figured that this page is mainly for robots, so shouldn't matter too much. Happy to add a fallback if folks disagree, or we could update this page later to use the upcoming Protocol grid mixins.

**Comment by alexgibson** [comment]
Ok, I'm going to say this is now ready for code review, removing the dnm label.

File: bedrock/mozorg/templates/mozorg/locales.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+{% block content %}
+<main role="main" class="mzp-l-content">
+  <header class="c-simple-header">
+    <h1 class="c-simple-header-title">Choose your language</h1>
+  </header>
+  <section class="c-block-list">
+    <h2 class="c-block-list-title">America</h2>
+    <ul>
+      <li><a href="/cak/">Maya Kaqchikel</a></li>
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/locales.html]: Each language should have a `lang` attribute associated with it, you can put it on the `<li>` or the `<a>`.

File: media/css/mozorg/locales.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/mozorg/locales.scss]: 👍

File: bedrock/mozorg/templates/mozorg/locales.html (hunk 2 above)
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/locales.html]: `<li lang="en-CA">`

Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/locales.html]: Great suggestion, will do!

**Comment by alexgibson** [comment]
Thanks @stephaniehobson updated

**Comment by stephaniehobson** [comment]
R+ 🌎

**Pull request merged**